deltachat-ffi/deltachat.h

```diff
@@ -6086,6 +6092,9 @@ void dc_event_unref(dc_event_t* event);
  *
  * There is no extra #DC_EVENT_MSGS_CHANGED event send together with this event.
  *
+ * If the message is an webxdc info message,
+ * dc_msg_get_parent() returns the webxdc instance the notification belongs to.
```

Comment:
I am a bit confused, didn't this already existed in the past? wasn't something like this what we use to get the parent webxdc message to show the app icon in the in-chat info-message?

Reply:
yip, this is not new.

but new is that webxdc info messages get notified, so i thought that is a nice additional information.

Reply:
(sorry I somehow read it as a header declaration 🥴 brain glitch)